### PR TITLE
Spec: Specify ruling outcomes

### PR DESCRIPTION
**Pull request by john-light**
- Specify the outcome if a dispute is ruled "Refuse to vote"  
- Specify the ruling priority in case of a tie

### CODE REVIEW

File: implementation-spec.md [hunk 1]
```diff
@@ -35,7 +35,12 @@ Apps installed on the organization and their parameters.
       - Arbitrator: `0xee4650cbe7a2b23701d416f58b41d8b76b617797`  
       - Staking pool: TBD  
       - Title: Aragon Network Agreement  
-      - Content: https://github.com/aragon/agreement/pull/1/files
+      - Content: https://github.com/aragon/agreement/pull/1/files  
+      - Refuse to rule outcome: Block
```
Comment (john-light) [inline implementation-spec.md]: This line should actually be a parameter (currently hard-coded) in the Disputable Voting app.

File: implementation-spec.md [hunk 2]
```diff
@@ -27,27 +27,33 @@ Apps installed on the organization and their parameters.
...
 - Agreement. Establishes human-readable guidelines that actions and proposals in the organization must comply with. Actors and proposal authors could have collateral that they have staked slashed if their actions/proposals do not comply with the Agreement.
 
     - Parameterization:  
       - Arbitrator: `0xee4650cbe7a2b23701d416f58b41d8b76b617797`  
       - Staking pool: TBD  
       - Title: Aragon Network Agreement  
-      - Content: https://github.com/aragon/agreement/pull/1/files
+      - Content: https://github.com/aragon/agreement/pull/1/files    
+      - Ruling priority:  
+        - 1: Refuse to rule (hard-coded in Aragon Court)
```
Comment (john-light) [inline implementation-spec.md]:
```suggestion
        - 1: Refuse to vote (hard-coded in Aragon Court)  
```

File: implementation-spec.md [hunk 3]
```diff
@@ -27,27 +27,33 @@ Apps installed on the organization and their parameters.
...
+        - 1: Refuse to vote (hard-coded in Aragon Court)  
+        - 2: Block  
+        - 3: Allow  
 
     - Disputable Voting 1 parameterization:  
       - Challenge response duration: 24 hours  
       - Collateral token: `0x960b236a07cf122663c4303350609a66a7b288c0`  
       - Action collateral: 100 ANT  
       - Challenge collateral: 100 ANT
+      - Refuse to rule outcome: Block
```
Comment (john-light) [inline implementation-spec.md]:
```suggestion
      - `Refuse to vote` outcome: Block
```

File: implementation-spec.md [hunk 4]
```diff
@@ -27,27 +27,33 @@ Apps installed on the organization and their parameters.
...
       - Action collateral: 100 ANT  
       - Challenge collateral: 100 ANT
+      - `Refuse to vote` outcome: Block
 
     - Disputable Voting 2 parameterization:  
       - Challenge response duration: 24 hours  
       - Collateral token: `0x960b236a07cf122663c4303350609a66a7b288c0`  
       - Action collateral: 10,000 ANT  
       - Challenge collateral: 1,000 ANT
+      - Refuse to rule outcome: Block
```
Comment (john-light) [inline implementation-spec.md]:
```suggestion
      - `Refuse to vote` outcome: Block
```

File: implementation-spec.md [hunk 5]
```diff
@@ -72,6 +78,7 @@ Apps installed on the organization and their parameters.
         - Collateral token: `0x960b236a07cf122663c4303350609a66a7b288c0`  
         - Action collateral: 100 ANT  
         - Challenge collateral: 100 ANT
+        - Refuse to rule outcome: Block
```
Comment (john-light) [inline implementation-spec.md]:
```suggestion
        - `Refuse to vote` outcome: Block
```

File: implementation-spec.md [hunk 6]
```diff
@@ -90,6 +97,7 @@ Apps installed on the organization and their parameters.
         - Collateral token: `0x960b236a07cf122663c4303350609a66a7b288c0`  
         - Action collateral: 10,000 ANT  
         - Challenge collateral: 1,000 ANT
+        - Refuse to rule outcome: Block
```
Comment (john-light) [inline implementation-spec.md]:
```suggestion
        - `Refuse to vote` outcome: Block
```

**Review by facuspagnuolo — APPROVED** [review]
LGTM!

**Pull request merged**